feature/experimental/p4rt/wbb.p4info.pb.txt

```diff
@@ -737,28 +156,11 @@ controller_packet_metadata {
   metadata {
     id: 3
     name: "unused_pad"
-    bitwidth: 7
+    annotations: "@padding"
+    bitwidth: 6
```

Comment:
Hi Nick! This is going to break P4RT tests for some vendors.

We need some time such that vendors can implement the change before we can merge it here.

There is an existing PR for this change: https://github.com/openconfig/featureprofiles/pull/2146

Reply:
Ah i see. I wasnt aware of the existing PR. I'll close this one then.